test/verify/check-storage-stratis

```diff
@@ -0,0 +1,370 @@
+
+        # Add a data blockdev
+        b.click('#detail-sidebar .pf-c-card__actions button')
+        self.dialog({'passphrase': "foodeeboodeebar",
+                     'disks': {dev_2: True}})
+        b.wait_in_text('#detail-sidebar', dev_2)
+        b.wait_in_text('#detail-sidebar .sidepanel-row:contains(%s)' % dev_2, "data")
+
+        # Reboot
+        m.spawn("sync && sync && sync && sleep 0.1 && reboot", "reboot")
+        m.wait_reboot()
+        m.start_cockpit()
+        b.relogin()
+        b.enter_page("/storage")
+        b.wait_visible("#storage-detail")
+
+        b.wait_in_text('#detail-header', "Locked Stratis Pool")
+        b.wait_in_text('#detail-header', dev_1)
+        b.wait_in_text('#detail-header', dev_2)
+
+        # Unlock the pool
+        b.click('#detail-header button:contains(Unlock)')
+        self.dialog({'passphrase': "foodeeboodeebar"})
+        b.wait_not_in_text('#detail-header', "Locked Stratis Pool")
+        b.wait_in_text('#detail-header', "pool0")
+
+        # Mount the filesystem
+        self.content_row_action(1, "Mount")
+        self.dialog({})
+        self.wait_mounted(1, 1)
+
+        # Reboot (this requires the passphrase)
+        self.setup_systemd_password_agent("foodeeboodeebar")
+        m.spawn("sync && sync && sync && sleep 0.1 && reboot", "reboot")
+        m.wait_reboot()
+        m.start_cockpit()
+        b.relogin()
+        b.enter_page("/storage")
+        b.wait_visible("#storage-detail")
+
+        # Filesystem should be mounted now
+        self.wait_mounted(1, 1)
+
+        # Destroy the pool
+        b.click('#detail-header button:contains(Delete)')
+        self.confirm()
+        b.wait_visible("#storage")
+        b.wait_not_in_text("#devices", "pool0")
+
+        # Check that the entry has disappeared from fstab
+        self.assertEqual(m.execute("grep /run/fsys1 /etc/fstab || true"), "")
+
+    def testCli(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        dev_1 = "/dev/sda"
+        m.add_disk("4G", serial="DISK1")
+        b.wait_in_text("#drives", dev_1)
+
+        dev_2 = "/dev/sdb"
+        m.add_disk("4G", serial="DISK2")
+        b.wait_in_text("#drives", dev_2)
+
+        # Create a pool outside of Cockpit
+        m.execute("stratis pool create TEST1 %s %s" % (dev_1, dev_2))
+        b.wait_in_text("#devices", "TEST1")
+        b.wait_in_text("#devices", "/dev/stratis/TEST1/")
+        b.click('.sidepanel-row:contains("TEST1")')
+        b.wait_visible("#storage-detail")
+        b.wait_in_text("#detail-sidebar", dev_1)
+        b.wait_in_text("#detail-sidebar", dev_2)
+
+        # Create two filesystems outside of Cockpit
+        m.execute("stratis filesystem create TEST1 fsys1")
+        b.wait_in_text("#detail-content", "fsys1")
+        m.execute("stratis filesystem create TEST1 fsys2")
+        b.wait_in_text("#detail-content", "fsys2")
```

Comment:
Can you check mounting/unmounting as well?

Reply:
Done